client/state/invites/test/selectors.js

```diff
@@ -3,6 +3,8 @@
  * External dependencies
  */
 import { expect } from 'chai';
+import sinon from 'sinon';
+import lodash from 'lodash';
```

Comment:
cc @dmsnell re [`lib/impure-lodash`](https://github.com/Automattic/wp-calypso/tree/master/client/lib/impure-lodash) removal - it looks like this approach + manipulating the `lodash` object as desired works just fine in the test environment.

Reply:
:+1:

Reply:
drive-by uninformed question: can we make this work with jests's [spyOn](https://facebook.github.io/jest/docs/en/jest-object.html#jestspyonobject-methodname) instead of sinon

Reply:
I used `sinon` here because (I think) using `jest.spyOn` and inspecting its calls would require converting these tests to Jest's `expect` instead of the current one.  As such, it seems better suited for a separate PR.